kibble.json

```diff
@@ -106,5 +184,6 @@
       "datasource": "Collection",
       "pageSize": 0
     }
-  ]
+  ],
+  "coreTemplateVersion": "0.4.2"
```

Comment:
Is this PR the only reason why we add this to `kibble.json`? I'd prefer we right something to the [site JS dir](https://github.com/shift72/core-template/tree/AB%236706-add-core-template-version-to-head/site/static/js) to `console.log()` info than show in `<head>` so we don't have to add more things to `kibble.json`.

Reply:
There's a few places that need `coreTemplateVersion` from `kibble.json`:

- Local kibble uses it in local site builds to put in the <head>.
- Local kibble sends it as a param to the site templates end point during a deploy.  The site templates end point adds it to the site template database record here.
- S3 kibble uses it in the live site builds to put in the <head>.

Putting `coreTemplateVersion` in `kibble.json` kills all of those birds with one stone.  I had originally planned to add it as a command line arg for the publish cobra command in kibble but that is messier because the builder worker has to manually concatenate the command if the arg exists.

I think `kibble.json` is a fine place to put this because the template is the source of truth for this info (not the apis/db/whatever).

If you would prefer to `console.log` `coreTemplateVersion` instead of write it in the <head>, we can do that... but it still needs to go in `kibble.json` or through the publish cobra command to be passed to the site templates end point.

Reply:
What are the missing words at the end of these sentences?

> Local kibble uses it in local site builds to put in the .
> S3 kibble uses it in the live site builds to put in the .

This might win on easiness but it still feels that it pollutes kibble with a concept it shouldn't care about.

Reply:
Damn what the?!?

Reply:
Huh... I guess Github was stripping out my unescaped `<head>` tags lol.  Edited the comment above!